src/components/commons/Comment/Comment.tsx

```
@@ -1,158 +1,202 @@
-        </div>
-        :
+    <div css={container(parentCommentId !== null, isDeleted ?? false)}>
+      {isDeleted ? (
+        <DeletedComment />
+      ) : (
         <>
-          {!isEditing ?
+          {!isEditing ? (
             <>
               <div css={profileContainer}>
                 <img css={profileImageStyle} src={roleImages[role]} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)